library/intel/adi_jesd204/adi_jesd204_hw.tcl

```diff
@@ -238,6 +252,8 @@ proc jesd204_validate {{quiet false}} {
     return false
   }
 
+  set_parameter_property BONDING_CLOCKS_EN VISIBLE [expr $num_of_lanes > 6]
```

Comment:
If I'm not mistaken bonded configuration makes sense just for TX. Can we hide the parameter if the current configuration is RX?

Reply:
Good remark. Yes, bonding configuration makes sense only for Tx. You can hide the option in Rx mode.